src/saver.ts

```diff
@@ -244,6 +244,19 @@
         }
     }
 
+    // TODO: Rename function
+    private async tempFunctionName(n: number): Promise<string> {
```

Comment:
The `async` keyword is unnecessary since there are no `await` statements inside the function.